### DISCUSSION TITLE
Add configuration for virtualized Combobox options

### DISCUSSION THREAD

**Discussion by vankhoawin** [twilio-labs/paste#1910] [Ideas] [answered]
Hey team,

The Combobox virtualized options performs great. One small thing we've run into however is that the [virtualized option height is statically defined](https://github.com/twilio-labs/paste/blob/45c24b01aeabf3db34868c4f0ae91b49a0493169/packages/paste-core/components/combobox/src/Combobox.tsx#L72-L81); would it be possible to expose these props or allow consumers to pass in their own `useVirtual` props? For our use-case, this would allow us to dynamically calculate the option height based on the option string length.

![image](https://user-images.githubusercontent.com/6936736/136105231-a5828756-b0dd-4834-ac9e-68e3d3b7a5da.png)

**Comment by andioneto** [comment, marked as answer]
Hey @vnguyen94 👋

🐛 We dug into it a bit and we found a [small bug in the combobox implementation](https://github.com/twilio-labs/paste/blob/main/packages/paste-core/components/combobox/src/ComboboxItems.tsx#L49) that is causing the virtualized options to not be rendered at the correct height.

🎫  We have created a ticket for it and are pulling it into our next sprint. We will follow up with you when it is released and fixed.

🧠 We will spend some time as a team discussing allowing additional configuration of the virtualization, and will get back to you about that in about a weeks' time.

Thanks again for all your help to improve paste!

**Reply by vankhoawin** [reply]
And thank you all for improving it!! 🏅

**Reply by gloriliale** [reply]
The fix for this is merged! 🎉
